web-app/js/portal/cart/Downloader.js

```diff
@@ -113,13 +113,25 @@ Portal.cart.Downloader = Ext.extend(Ext.util.Observable, {
         });
     },
 
-    _onAsyncDownloadRequestSuccess: function() {
+    _onAsyncDownloadRequestSuccess: function(response) {
+        var serviceMessage = "";
+        if (this.params.serviceResponseHandler) {
+            serviceMessage = this.params.serviceResponseHandler(response.responseText);
+        }
```

Comment:
I think you might have intended to replace this with a call to _getServicemessage()?